### PR TITLE
Migrate frontend into monorepo

### PR DESCRIPTION
**Pull request by jrjohnson**
For now the only repo within this monorepo is the frontend itself, but later we'll be moving everything else in as well. For now get it working alone.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Netlify issues will fix after merging and Percy failures are due to this being the first time Percy has been working in a while.

**Review by stopfstedt — APPROVED** [review]
LGTM, but i'm holding this back due to some routing issues coming out of common.

**Comment by stopfstedt** [comment]
unblocked from merging again after further discussion with @jrjohnson. we'll address the mentioned issues as follow-up.

**Pull request merged**
